mysite/blog/templates/pagination.html

```diff
@@ -1,13 +1,13 @@
 <div class="pagination">
  <span class="step-links">
  {% if page.has_previous %}
- <a href="?page={{ page.previous_page_number }}">Previous</a>
+   <a href="?page={{ page.previous_page_number }}">Previous</a>
  {% endif %}
- <span class="current">
+   <span class="current">
  Page {{ page.number }} of {{ page.paginator.num_pages }}.
  </span>
- {% if page.has_next %}
- <a href="?page={{ page.next_page_number }}">Next</a>
- {% endif %}
+   {% if page.has_next %}
+     <a href="?page={{ page.next_page_number }}">Next</a>
+   {% endif %}
  </span>
-</div>
\ No newline at end of file
+</div>

```

Comment:
This should not be part of this PR. Push is formating change directly to master.